zrml/swaps/src/tests.rs

```diff
@@ -31,17 +29,6 @@ const _100: u128 = 100 * BASE;
 const _101: u128 = 101 * BASE;
 const _105: u128 = 105 * BASE;
 
-#[test]
-fn rpc_serialization_deserialization_do_not_panic() {
-    ExtBuilder::default().build().execute_with(|| {
-        let runtime = Runtime;
-        let block_id = BlockId::Number(0);
-        let _ = runtime.get_spot_price(&block_id, 1, Asset::PoolShare(0), Asset::PoolShare(1));
-        let _ = runtime.pool_account_id(&block_id, 1);
-        let _ = runtime.pool_shares_id(&block_id, 1);
-    });
-}
-
```

Comment:
So the test doesn't work?

Reply:
Thought this test could catch the bad serialization error but it doesn't and it probably won't because the real issues lies when interacting with nodes, not the runtime.
```
User --- (calls) ---> Node --- (calls) ---> Runtime
```
Therefore, it isn't testing anything useful. Nevertheless, I can rollback this removal if desirable.

Reply:
No let's remove it since it doesn't test what it should be testing. We will need to test this using an integration test in the JS libraries.